tidb-cloud/limited-sql-features.md

```diff
@@ -121,114 +121,114 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 | `mysql` | `gc_delete_range_done` | Not supported [^4] | Not supported [^4] |
 | `mysql` | `opt_rule_blacklist` | Not supported [^4] | Not supported [^4] |
 | `mysql` | `tidb` | Not supported [^4] | Not supported [^4] |
-| `mysql` | `tidb_ttl_job_history` | Supported | Not supported [^9] |
```

Comment:
Since TTL is supported on all products, we can simply remove these lines.

Reply:
resolved